app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  after_create :update_likes_counter
```

Comment:
- [OPTIONAL] Good job so far but I think it would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -`after_save` instead of `after_create` to call the `#update_likes_counter` method every time a new post has been saved to the database 🙃 🙃